Week-1/html/exercise_1.2/music_player/css/currentsong.css

```diff
@@ -0,0 +1,37 @@
+.album-wrapper{
+  display:flex;
+  background-color: #454545;
+  padding: 24px;
+}
+
+.album-details{
+  display:flex;
+  flex-direction:column;
+  color: white;
+  justify-content: space-between;
+  font-size: x-small;
+  font-weight: bold;
+}
+
+.album-detail-wrapper{
+  
+}
+
+.album-controls{
+  display:flex;
+  justify-content: space-between;
+}
+
+.album-control-wrapper{
+  display: flex;
+  column-gap: 6px;
+}
+
+.album-control-label{
```

Comment:
what is use of this style, if it's not represent any thing ?

Reply:
As per the screenshot I have de-composed the UI into small units and accordingly given the class name for CSS to avoid using inline CSS.

Reply:
As per the screenshot I have de-composed the UI into small units and accordingly given the class name for CSS to avoid using inline CSS.